applications/nrf5340_audio/src/modules/lc3_playback.c

```diff
@@ -0,0 +1,228 @@
+#include "lc3_playback.h"
+#include <stdint.h>
+#include <zephyr/sys/ring_buffer.h>
+#include "sd_card.h"
+#include "hw_codec.h"
+#include "sw_codec_lc3.h"
+#include "sw_codec_select.h"
+#include "pcm_stream_channel_modifier.h"
+#include "pcm_mix.h"
+#include "audio_i2s.h"
+#include "sw_codec_lc3.h"
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(lc3_playback, 4); /* For draft PR feedback: going to change this line */
+
+#define RING_BUF_SIZE		1920 /* This can be modified */
```

Comment:
Would be great if we add something in name for indicating this is for specific usage.